src/dimension_slice.c

```diff
@@ -793,7 +795,7 @@ dimension_slice_check_is_chunk_uncompressed_tuple_found(TupleInfo *ti, void *dat
 	foreach (lc, chunk_ids)
 	{
 		int32 chunk_id = lfirst_int(lc);
-		if (ts_chunk_can_be_compressed(chunk_id))
+		if (ts_chunk_can_be_compressed_or_reordered(chunk_id))
```

Comment:
Shouldn't combine the 2 different policies together. This is only for compress chunk job. Need functionality specifically for reorder chunks. (similar to ts_dimension_slice_get_chunkid_to_compress )

Reply:
Actually it's not a combination of policies, it's just a check shared among both policies. Both policies require the chunk not to be compressed or dropped. I don't think it makes sense to clone `ts_chunk_can_be_compressed `into `ts_chunk_can_be_reordered` 'cause the code will be duplicated. Maybe rename this check to `ts_chunk_is_not_compressed_or_dropped`?

Reply:
Could you revert this change?

Reply:
Is it OK to call `ts_chunk_can_be_compressed` from `dimension_slice_check_chunk_stats_tuple_found` which is used only for reorder policy? Or I should create a separate function `ts_chunk_can_be_reordered`?

Reply:
I don't think we need another function since this is just a check for uncompressed chunk. (The other option is to rename ts_chunk_can_be_compressed as you did. But would prefer to keep the changes minimal.)